selenium/support/utils.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Builder } from 'selenium-webdriver';
+import chrome from 'selenium-webdriver/chrome';
+import fetch from 'node-fetch';
+import { BASE_URL, BASE_SERVER_URL } from './constants';
+
+const username = 'admin';
+const password = 'admin';
+let isAuthEnabled = false;
+let authToken = null;
+
+export const buildChromeDriver = () => {
+  const options = new chrome.Options()
+  
+  options.addArguments('--disable-dev-shm-usage')
+  options.addArguments('--no-sandbox')
+  // options.addArguments('--headless')
+
+  return new Builder()
+    .forBrowser('chrome')
+    .setChromeOptions(options)
+    .build()
+}
+
+export const loginIfRequired = async (driver) => {
```

Comment:
I wasn't able to fully test this function (particularly the block starting at line 50). Was not able to simulate the login process locally. On paper it should work, but we might see failures when logging in is actually required.